docs/generate-evidence-strings.md

```diff
@@ -0,0 +1,279 @@
+  + Filter this file, extracting only records with allowed levels of clinical significance (as provided by ClinVar). For example, this step filters out records where the clinical significance is “Benign”, meaning that the variant *does not* contribute to a disease. Result is `clinvar.filtered.json.gz`.
+* **Map variants to gene and functional consequences.** Each evidence string must include the variant, the gene it affects, and the functional effect it has in that gene. We obtain this information separately for repeat expansion variants and for all other types. Detailed information can be found in the [repository where the pipelines are contained](https://github.com/EBIvariation/vep-mapping-pipeline).
+  + Run repeat expansion pipeline
+  + Run main VEP pipeline
+  + Concatenate the results into a single file
+* **Generate the evidence strings.** Here, we integrate all of the information produced to generate the evidence strings. This outputs multiple files:
+  + `evidence_strings.json` — evidence strings for submitting to Open Targets
+  + `eva_clinvar.txt` — ZOOMA feedback with trait names
+* Prepare ClinVar xRefs ZOOMA feedback (see more on that below)
+
+```bash
+cd ${CODE_ROOT} && \
+${BSUB_CMDLINE} -K -n 8 -M 16G \
+  -o ${BATCH_ROOT}/logs/convert_clinvar_files.out \
+  -e ${BATCH_ROOT}/logs/convert_clinvar_files.err \
+  java -Xmx15G -jar ${CODE_ROOT}/clinvar-xml-parser/target/clinvar-parser-1.0-SNAPSHOT-jar-with-dependencies.jar \
+  -i ${BATCH_ROOT}/clinvar/ClinVarFullRelease_${CLINVAR_RELEASE}.xml.gz \
+  -o ${BATCH_ROOT}/clinvar && \
+${BSUB_CMDLINE} -K \
+  -o ${BATCH_ROOT}/logs/filter_clinvar_json.out \
+  -e ${BATCH_ROOT}/logs/filter_clinvar_json.err \
+  python3 ${CODE_ROOT}/bin/clinvar_jsons/extract_pathogenic_and_likely_pathogenic_variants.py \
+  -i ${BATCH_ROOT}/clinvar/clinvar.json.gz \
+  -o ${BATCH_ROOT}/clinvar/clinvar.filtered.json.gz && \
+${BSUB_CMDLINE} -K \
+  -o ${BATCH_ROOT}/logs/consequence_repeat_expansion.out \
+  -e ${BATCH_ROOT}/logs/consequence_repeat_expansion.err \
+  python3 ${CODE_ROOT}/vep-mapping-pipeline/run_repeat_expansion_variants.py \
+  --clinvar-summary-tsv ${BATCH_ROOT}/clinvar/variant_summary_${CLINVAR_RELEASE}.txt.gz \
+  --output-consequences ${BATCH_ROOT}/gene_mapping/consequences_1_repeat.tsv \
+  --output-dataframe ${BATCH_ROOT}/gene_mapping/repeat_dataframe.tsv && \
+${BSUB_CMDLINE} -K \
+  -o ${BATCH_ROOT}/logs/consequence_vep.out \
+  -e ${BATCH_ROOT}/logs/consequence_vep.err \
+  bash ${CODE_ROOT}/vep-mapping-pipeline/run_consequence_mapping.sh \
+  ${BATCH_ROOT}/clinvar/clinvar_${CLINVAR_RELEASE_YEAR}${CLINVAR_RELEASE_MONTH}??.vcf.gz \
+  ${BATCH_ROOT}/gene_mapping/consequences_2_vep.tsv && \
+cat \
+  ${BATCH_ROOT}/gene_mapping/consequences_1_repeat.tsv \
+  ${BATCH_ROOT}/gene_mapping/consequences_2_vep.tsv \
+  > ${BATCH_ROOT}/gene_mapping/consequences_3_combined.tsv && \
```

Comment:
Thank you for finding this. This change actually is not because of this pull request — the file was renamed in https://github.com/EBIvariation/eva-cttv-pipeline/pull/94 and I didn't update the spreadsheet commands. Now fixed.

Reply:
Additionally, something _this_ PR would break is a command to calculate the “Number of distinct mapped trait names”. I have updated it in the spreadsheet, but also kept the old command for now.

Also, added a separate metric for the number of repeat expansion variants mappings (this batch will be the first one to include them)